lib/active_meta/attribute.rb

```diff
@@ -31,9 +31,7 @@ def overload(&block)
     end
 
     def register_rule(rule)
-      unless rule.is_a? ActiveMeta::Rule
-        raise ArgumentError, "no rule given for attribute #{@attribute}"
-      end
+      raise ArgumentError, "no rule given for attribute #{@attribute}" unless rule.is_a? ActiveMeta::Rule
```

Comment:
I use inline condition as much as possible. But maybe it's not good for the clarity of the code ?